GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -166,6 +166,9 @@ void Mesh::generateVAO() {
         return;
     }
 
+    GLuint vaoID_ = 0;
+	GLuint triangle_vboID_, vert_vboID_, norm_vboID_, tex_vboID_;
```

Comment:
I don't know if you intend this indenting to signify anything ... if so, it won't survive a Shift+Ctrl+F